src/orquestra/sdk/_base/_log_adapter.py

```diff
@@ -8,31 +8,45 @@
 import json
 import logging
 import os
-from typing import Tuple
+import typing as t
+from datetime import datetime, timezone
 
 from orquestra.sdk.schema.workflow_run import TaskRunId, WorkflowRunId
 
-RAY_LOGS_FORMAT = '{"timestamp": "%(asctime)s", "level": "%(levelname)s", "filename": "%(filename)s:%(lineno)s", "message": %(message)s}'  # noqa
-logging.basicConfig(
-    format=RAY_LOGS_FORMAT,
-)
+# NOTE: the `message`, `wf_run_id`, and `task_run_id` value placeholders don't come with
+# "" quotes. We already add them when we json.dumps() the value. This ensure proper JSON
+# escaping and handling null values.
+FORMAT = '{"timestamp": "%(asctime)s", "level": "%(levelname)s", "filename": "%(filename)s:%(lineno)s", "message": %(message)s, "wf_run_id": %(wf_run_id)s, "task_run_id": %(task_run_id)s}'  # noqa
 
 
 class TaggedWorkflowTaskLogger(logging.LoggerAdapter):
     """
     Adding a workflow/task context to log outputs.
     """
 
-    def process(self, msg, kwargs):
-        return (
-            json.dumps(
-                {
-                    "run_id": f'{self.extra["workflow_run_id"]}@{self.extra["task_run_id"]}',  # noqa: E501
-                    "logs": msg,
-                }
-            ),
-            kwargs,
-        )
+    def process(
+        self, msg: str, kwargs: t.Mapping[str, t.Any]
+    ) -> t.Tuple[str, t.MutableMapping[str, t.Any]]:
+        """
+        Args:
+            msg: message passed to the standard logger method calls.
+            kwargs: custom kwargs passed to the standard logger method calls.
+        """
+
+        # Ensure the message can be stored as a JSON. This will escape any
+        # JSON-forbidden chars.
+        new_msg = json.dumps(msg)
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**question:** ‏I assume msg is always guaranteed to be a string because it comes after any `f"{f_string}"` or `"formatted string {}".format(object())`?

Reply:
Good catch! I'll wrap it in `str()` so we don't have to guess.